src/entities.rs

```diff
@@ -106,6 +106,9 @@ pub struct MediaEntity {
     ///For media entities corresponding to videos, this contains extra information about the linked
     ///video.
     pub video_info: Option<VideoInfo>,
+    ///Alt text for accessibility. If alt text isn't available for a given image, this will be
+    ///`None`.
```

Comment:
Docs suggestion:

> Media alt text, if present.

I've seen alt text attached to videos, so it's probably best not to assume it'll only be attached to images.